test/diameter_avp_filter_SUITE.erl

```diff
@@ -188,7 +188,7 @@ simple_path(_Config) ->
     ?equal(undefined, maps:get('3GPP-IMSI', Before, undefined)),
 
     #{'Service-Information' := [#{'PS-Information' := [PSInfoAfter]}]} = After,
-    ?equal(false, is_map_key('3GPP-MS-TimeZone', PSInfoAfter)),
+    ?equal(undefined, maps:get('3GPP-MS-TimeZone', PSInfoAfter, undefined)),
```

Comment:
There is maps:is_key/2 for checking keys.
maps:get/3 works, but is slightly less clear.

Reply:
True. This change is a workaround for a compiler bug in the OTP 22 versions we use in CI, that fails on the boolean check.